rasa/cli/interactive.py

```diff
@@ -23,8 +29,28 @@ def add_subparser(
     train.add_domain_param(interactive_parser)
     train.add_joint_parser_arguments(interactive_parser)
     _add_interactive_arguments(interactive_parser)
+
     interactive_parser.set_defaults(func=interactive)
 
+    interactive_subparsers = interactive_parser.add_subparsers()
+    interactive_core_parser = interactive_subparsers.add_parser(
+        "core",
+        conflict_handler="resolve",
+        parents=parents,
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+        help="Train a Rasa Core model with interactive learning",
+    )
+
+    train.add_domain_param(interactive_core_parser)
+    core_cli.add_general_args(interactive_core_parser)
+    train.add_stories_param(interactive_core_parser)
+    train.add_domain_param(interactive_core_parser)
+    run.add_run_arguments(interactive_core_parser)
+    _add_interactive_arguments(interactive_core_parser)
+    train.add_general_arguments(interactive_core_parser)
```

Comment:
why is it so many? Should it not be the same as for the other `interactive` command?

Reply:
The `interactive` command trains a stack model whereas the `interactive core` command only trains a core model. As training of a core model expects different kind of parameters, the arguments differ.